### DISCUSSION TITLE
[Feature Request] [combobox] make clearing on [escape] optional

### DISCUSSION THREAD

**Discussion by erikkroes** [ing-bank/lion#1002] [General]
## Expected behavior

I would expect 'escape' to close the dropdown but leave the text in the textbox.

## Actual Behavior

'Escape' empties the textbox. This behaviour is optional. I would follow the default as to not confuse user( expectation)s.

**Comment by tlouisse** [comment]
It currently follows aria spec, but making those configurable would be nice imo 👍 
I will move this to discussions as a feature request, alright?
